docs/src/lateral/local-inertial.md

```diff
@@ -0,0 +1,83 @@
+of the local inertial approximation on a staggered grid is as follows (Bates et al., 2010):
+
+```math
+Q_{t+\Delta t} = \frac{Q_t - g A_t \Delta t S_t}{(1+g\Delta t n^2 |Q_t| / (R_t^{4/3} A_t))}
+``` 
+where ``Q_{t+\Delta t}`` is the river flow [m``^3``/s] at time step ``t+\Delta t``, ``g`` is
+acceleration due to gravity [m/s``^2``], ``A_t`` is the cross sectional flow area at the
+previous time step, ``R_t`` is the hydraulic radius at the previous time step, ``Q_t`` is
+the river flow [m``^3``/s] at the previous time step, ``S_t`` is the water surface slope at
+the previous time step and ``n`` is the Manning's roughness coefficient [m``^{-1/3}`` s].
+
+The momentum equation is applied to each link between two river grid cells, while the
+continuity equation over ``\Delta t`` is applied to each river cell:
+
+```math
+h^{t+\Delta t} = h^t + \Delta t \frac{Q^{t+\Delta t}_{src} - Q^{t+\Delta t}_{dst}}{A}
+``` 
+where ``h^{t+\Delta t}`` is the water depth [m] at time step ``t+\Delta t``, ``h^t`` is the
+water depth [m] at the previous time step, ``A`` is the river area [m``^2``] and ``Q_{src}``
+and ``Q_{dst}`` represent river flow [m``^3``/s] at the upstream and downstream link of the
+river cell, respectively.
+
+The model time step ``\Delta t`` for the local inertal model is estimated based on the
+Courant-Friedrichs-Lewy condition (Bates et al., 2010):
+
+```math
+\Delta t = min(\alpha \frac{\Delta x_i}{\sqrt{(gh_i)}})
+```
+
+where ``\sqrt{(gh_i)}`` is the wave celerity for river cell ``i`` , ``\Delta x_i`` is the
+river length [m] for river cell ``i`` and ``\alpha`` is a coefficient (typically between 0.2
+and 0.7) to enhance the stability of the simulation.
+
+In the TOML file the following properties related to the local inertial model can be
+provided for the `sbm` model type:
+
+```toml
+[model]
+river_routing = "local-inertial" # default is kinematic-wave
+inertial_flow_alpha = 0.5      # alpha coefficient for model stability (default = 0.7)
+froude_limit = true            # default is true, limit flow to subcritical-critical according to Froude number
+h_thresh = 0.1                 # water depth [m] threshold for calculating flow between cells (default = 1e-03)
+riverlength_bc = 1000          # river length [m] for boundary points (default = 1e05)
+```
+
+The momentum equation is most stable for low slope enviroments, and to keep the simulation
+stable for (partly) steep environments the `froude_limit` option can be set to true. This
+setting limits flow to subcritical-critical conditions based on the Froude number ($\le 1$),
+similar to Coulthard et al. (2013) in the CAESAR-LISFLOOD model and Adams et al. (2017) in
+the Landlab v1.0 OverlandFlow component.
+
+For the downstream boundary condition (ghost point) a fixed water surface elevation of 0 m
+is used. The river length [m] of the boundary cell can be set through the TOML file with
+`riverlength_bc`, the default value is 100 km. 
```

Comment:
I think the default fixed water surface elevation of 0 m is fine (~sea level boundary). The `riverlength_bc` is a bit more tricky to generalize, left this default value out for now.

Reply:
Ok thanks. Searching for `riverlength_bc` in the code, I see you updated the actual default, but a few comments/docs still need to be updated as well:
```
# level). river length at boundary point is by default 1.0e5 m (riverlength_bc).
riverlength_bc = 1000 # river length [m] for boundary points (default = 1e05)
`riverlength_bc`, the default value is 100 km.
```

A default of 0 for the elevation seems reasonable for many applications, though I don't know if we have a highland model, or one that goes to the Dead Sea (-430m), how much that default value will affect the results. Defaults should generally be safe, otherwise they shouldn't be a default. Can't we copy the riverlength from upstream as well, and extrapolate the elevation?

Reply:
I think copying the riverlength from upstream as well, and extrapolate the elevation is more risky, since we don't know the (dynamic) river depth at the upstream cell. This can lead easily (especially with smaller grid cells and thus river lengths) to an overestimation of the water slope. I also discussed this with @DirkEilander earlier and we concluded then that this is a simple appropriate solution for now, similar to for example the implementation in CaMa-Flood.

Reply:
Thanks for catching! I did update the docs.

Reply:
Ok good! Glad to hear you considered the options.

Reply:
An alternative could be to set the water surface elevation of the ghost node based on the river bed elevation of the outlet cell. In that case we could use a fixed large default value for `riverlengt_bc` (say 10 or 100 km) such that it will not influence the upstream water levels independent of the elevation of the outlet. This bc basically simulates a zero water depth boundary condition at a set distance which is arguable more generic in light of the extreme cases that @visr pointed out.  I believe this is currently also used as one of the boundary condition types you can set in SFINCS.

Reply:
Thanks for the comments @DirkEilander, yes this is indeed a more generic approach, will update the code based on this.